app/models/katello/root_repository.rb

```diff
@@ -214,6 +221,20 @@ def ensure_valid_docker_tags_whitelist
       end
     end
 
+    def ensure_valid_required_tags
+      return if required_tags.all?(&:blank?)
```

Comment:
Should this be `required_tags.empty?`

Reply:
Needed to account for both `['']` and `[]` which this does in a single line.  (I think angular may be sending the former sometimes)

Reply:
I think [''] is invalid

Reply:
retested and I'm not sending any `['']` any more.  So changing to `.empty?`.